profiles/base.yaml

```diff
@@ -382,6 +382,7 @@ storage_containers:
 gear:
 gear_sets:
 empath_healing:
+saferoom_health_threshold: 25
```

Comment:
25 Seems way too high for this, you can have multiple bleeders and still be under 20.